### PR TITLE
fedekunze/fix BeginRedelegate tx

### PR DESCRIPTION
**Pull request by fedekunze**
`BeginRedelegate` tx was being shown as `Unknown Transaction Type`

### CODE REVIEW

File: tests/unit/__snapshots__/TmBlock.spec.js.snap [hunk 1]
```diff
@@ -409,16 +409,30 @@ exports[`TmBlock.vue has the expected html structure 1`] = `
...
-              <div class="tm-li-tx__content__information"></div>
+              <div class="tm-li-tx__content__information">
+                <div>
+                  From&nbsp;
+                  <router-link to="/cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctqzh8yqw">
+                    cool validator
+                  </router-link>
+                  to&nbsp;
+                  <router-link to="/cosmosvaladdr157mg9hnhchfrqvk3enrvmvj29yhmlwf759xrgw">
+                    cosmosvaladdr157mg9hnhchfrqvk3enrvmvj29yhmlwf759xrgw
```
Comment (faboweb) [inline tests/unit/__snapshots__/TmBlock.spec.js.snap]: Why is line 428 a moniker and this one an address?

Comment (fedekunze) [reply tests/unit/__snapshots__/TmBlock.spec.js.snap]: I guess that happens when the validator is not identified ? I'll double check

Comment (fedekunze) [reply tests/unit/__snapshots__/TmBlock.spec.js.snap]: Added to tests

File: tests/unit/__snapshots__/TmLiStakeTransaction.spec.js.snap [hunk 2]
```diff
@@ -112,7 +112,7 @@ exports[`TmLiStakeTransaction should show redelegations 1`] = `
           <router-link
             to="/validator/cosmosvaladdr157mg9hnhchfrqvk3enrvmvj29yhmlwf759xrgw"
           >
-            cosmosvaladdr157mg9hnhchfrqvk3enrvmvj29yhmlwf759xrgw
+            Kentucky val
```
Comment (faboweb) [inline tests/unit/__snapshots__/TmLiStakeTransaction.spec.js.snap]: Billy is with us :D

Comment (fedekunze) [reply tests/unit/__snapshots__/TmLiStakeTransaction.spec.js.snap]: @okwme 😃 👨‍🌾

**Pull request merged**